bin/gtk-pipe-viewer

```diff
@@ -1744,19 +1866,9 @@ sub show_comments_window {
 
     $feeds_title->set_markup("<big>$video_title</big>");
     $feeds_title->set_tooltip_markup("$video_title");
-
     $feeds_liststore->clear;
-
     $feeds_window->show;
-
-    Glib::Idle->add(
-        sub {
-            display_comments($yv_obj->comments_from_video_id($videoID));
-            return 0;
-        },
-        [],
-        Glib::G_PRIORITY_LOW
-    );
+    set_comments;
```

Comment:
We should prefer calling functions explicitly with `()`. Mainly because it's more readable and also because the code won't break if the function declaration is moved after the function call.

Example:

```perl
use 5.014;

test;        # error
#test();     # this works

sub test {
    say "here";
}
```